<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbSystem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>384</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 0, 255);</string>
    </property>
    <property name="title">
     <string>System</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lPort">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(0, 0, 0);
font: 700 14pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Port :-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePort">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);
font: 700 14pt &quot;Segoe Script&quot;;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pbStart">
         <property name="styleSheet">
          <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
         </property>
         <property name="text">
          <string>Listen</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lPort_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(0, 0, 0);
font: 700 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-style:italic;&quot;&gt;Enter Message&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="leData">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);
font: 700 14pt &quot;Segoe Script&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pbSend">
       <property name="styleSheet">
        <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>501</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>Console Data</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QListWidget" name="lwConsole">
       <property name="styleSheet">
        <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);</string>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pbClearConsole">
       <property name="styleSheet">
        <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>891</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string>Received Data</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QListWidget" name="lwReceived">
       <property name="styleSheet">
        <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
color: rgb(255, 0, 127);</string>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pbClearReceived">
       <property name="styleSheet">
        <string notr="true">font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbClearConsole</sender>
   <signal>clicked()</signal>
   <receiver>lwConsole</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>624</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClearReceived</sender>
   <signal>clicked()</signal>
   <receiver>lwReceived</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>723</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>618</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
